libs/base/ui/form/password/src/password-input.model.ts

```diff
@@ -30,3 +30,26 @@ export const enum PasswordVisibilityStrategy {
    */
   Hover = 'hover',
 }
+
+export interface ValidationOptions {
+  /**
+   * Minimum length the password must have
+   */
+  minLength?: number;
+  /**
+   * Maximum allowable length for the password
+   */
+  maxLength?: number;
+  /**
+   * Requires at least one uppercase letter in the password
+   */
+  requireUpperLetter?: boolean;
+  /**
+   * Requires at least one numeric character in the password
+   */
+  requireNumber?: boolean;
+  /**
+   * Requires at least one special character in the password
+   */
+  requireSpecialChar?: boolean;
+}
```

Comment:
What about:

```ts
export interface PasswordValidationOptions {
  /**
   * Specifies the minimum length required for the password.
   */
  minLength?: number;
  /**
   * Specifies the maximum allowable length for the password.
   */
  maxLength?: number;
  /**
   * Specifies the minimum number of uppercase characters required in the password.
   */
  minUppercaseChars?: number;
  /**
   * Specifies the minimum number of numeric characters required in the password.
   */
  minNumbers?: number;
  /**
   * Specifies the minimum number of special characters required in the password.
   */
  minSpecialChars?: number;
}
```

Makes it a little more flexible, we could default the values to `1`.

Reply:
Thanks, I like your idea with numbers, but I think it is better to left props without default values because of flexibility (e.g. use just minLength rule). What